docs/Install-Guide/Configure.md

```diff
@@ -81,8 +81,8 @@ Breaking this down into pieces:
 (the name is arbitrary, `gv0` was chosen simply because
 it’s less typing than `gluster_volume_0`).
 - make the volume a replica volume
-- keep a copy of the data on at least 2 bricks at any given time.
-Since we only have two bricks total, this
+- keep a copy of the data on at least 3 bricks at any given time.
```

Comment:
This doesn't seem right - we do keep data on at least 2.

Reply:
have a look at the command above the changed  lines. There "replica 3" is specified. thats the reason for my confusion:
Creating a volume with replica 3 over three bricks, doesn't make sense IMHO with the specified sentence.

Thats  why I suggest a change - this could as well also be, either
a) turning the command down to replica 2 or
b) xtending the documentation and explaining, why it makes sense.

Reply:
We have a replica 3, but in case of a single failure, we keep 2 copies and continue to function. This is why 'at least 2' is correct.

Reply:
Please read the whole paragraph from the perspective of a user, trying to understand:

The Command creates a "replica 3" over 3 bricks and then this command is explained.
In this context the sentence doesn't make sense, because it is not about failure-scenario.

OR: I have completely misunderstood, what replica 3 means. In my understanding "replica 3" means, that we try to replicate the data to at least 3 bricks.

Reply:
I understand, but I don't think your fix is correct. We could rephrase it to better explain the mechanism. It strives to keep 3 copies, but is still functioning with 2 copies.

Reply:
thats what I meant with option b above.